extensions/cornerstone/src/OHIFCornerstoneViewport.js

```diff
@@ -58,24 +49,24 @@ class OHIFCornerstoneViewport extends Component {
    * Obtain the CornerstoneTools Stack for the specified display set.
    *
    * @param {Object[]} studies
-   * @param {String} studyInstanceUid
+   * @param {String} StudyInstanceUID
    * @param {String} displaySetInstanceUid
-   * @param {String} [sopInstanceUid]
+   * @param {String} [SOPInstanceUID]
    * @param {Number} [frameIndex=1]
    * @return {Object} CornerstoneTools Stack
    */
   static getCornerstoneStack(
     studies,
-    studyInstanceUid,
+    StudyInstanceUID,
     displaySetInstanceUid,
-    sopInstanceUid,
+    SOPInstanceUID,
```

Comment:
Again, a little odd, but I get it. Not sure where we stand on caps

Reply:
After reviewing 130 files, I've warmed up to this. It feels odd, but it adds a lot of value I really, really like consistency; now we just need an ESLint rule. Some cases still seem fuzzy to me. Like a PatientName property that is a display representation of the same DICOM tag.

IE. W/ formatting applied, as part of search filter input

We also had a few cases of the same tag within the same lexical scope. This lead to one variable using this official naming format, and the other using a different name (and alternative casing).